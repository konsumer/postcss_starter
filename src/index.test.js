import initStoryshots from '@storybook/addon-storyshots'

// this just runs snapshots on all stories
initStoryshots()
