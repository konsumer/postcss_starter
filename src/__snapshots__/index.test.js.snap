// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Demo example 1`] = `
<div
  className="Demo"
>
  This is a demo-component. You can see the code in 
  <code>
    src/components/Demo
  </code>
  <button
    onClick={[Function]}
  >
    click me!
  </button>
</div>
`;
